### PR TITLE
DOC: Add none as a default argument for orient

### LINKED ISSUE #48252: DOC: read_json should list `None` for `orient`
### Pandas version checks

- [X] I have checked that the issue still exists on the latest versions of the docs on `main` [here](https://pandas.pydata.org/docs/dev/)

### Location of the documentation

https://pandas.pydata.org/docs/reference/api/pandas.read_json.html

### Documentation problem

`None` is an acceptable type for `orient`.

### Suggested fix for documentation

Add `None` to the list of acceptable arguments, or maybe just an `, optional`

### PR DESCRIPTION
**Pull request by ambujpawar**
- [x] closes #48252 

Add None as an acceptable argument for orient in read_json over [here](https://pandas.pydata.org/docs/reference/api/pandas.read_json.html)

### CODE REVIEW

**Review by phofl — CHANGES_REQUESTED** [review]
Can you add what None does and if it is allowed for typ=series or typ=frame?

**Comment by ambujpawar** [comment]
Based on my understanding, None does nothing. It is just an acceptable value. 
The value of 'orient' depends on the value of "typ" parameter as mentioned later in the docstring over [here](https://pandas.pydata.org/docs/reference/api/pandas.read_json.html):

> The allowed and default values depend on the value of the typ parameter.

> For Series: default is index and For Frame: default is columns

Since, it is already mentioned, I am not quite sure how shall I improve it. Shall I change the default value of it being 'columns' instead of None since the default value of 'typ' is 'Series'?

File: pandas/io/json/_json.py [hunk 1]
```diff
@@ -501,7 +501,7 @@ def read_json(
         By file-like object, we refer to objects with a ``read()`` method,
         such as a file handle (e.g. via builtin ``open`` function)
         or ``StringIO``.
-    orient : str
+    orient : str, default None
         Indication of expected JSON string format.
         Compatible JSON strings can be produced by ``to_json()`` with a
         corresponding orient value.
```
Comment (phofl) [inline pandas/io/json/_json.py:507]: ``The set of possible orients is:`` This read to me as None is not an option, hence please clarify

File: pandas/io/json/_json.py [hunk 2]
```diff
@@ -501,7 +501,7 @@ def read_json(
         By file-like object, we refer to objects with a ``read()`` method,
         such as a file handle (e.g. via builtin ``open`` function)
         or ``StringIO``.
-    orient : str
+    orient : str, default None
```
Comment (datapythonista) [inline pandas/io/json/_json.py:504]: I think in other docstrings we've been using `orient : str ,optional` for cases like this.

File: pandas/io/json/_json.py [hunk 3]
```diff
@@ -514,6 +514,7 @@ def read_json(
         - ``'index'`` : dict like ``{{index -> {{column -> value}}}}``
         - ``'columns'`` : dict like ``{{column -> {{index -> value}}}}``
         - ``'values'`` : just the values array
+        - None : orient will then be assigned based on `typ` parameter
```
Comment (datapythonista) [inline pandas/io/json/_json.py:517]: This seems to be redundant with the next paragraph, where it's explained what the default value of orient will be based on `typ`.  And IMHO, it makes things more confusing. I'd just remove this line and just leave the existing paragraph.

Comment (ambujpawar) [reply pandas/io/json/_json.py:517]: After reading it again, I agree with you completely. Removing it!!

File: pandas/io/json/_json.py (hunk 2 above)
Comment (ambujpawar) [reply pandas/io/json/_json.py:504]: I didn't know this. Thanks for the info.
Updating the PR shortly!!

**Review by datapythonista — APPROVED** [review]
Thanks @ambujpawar

**Review by phofl — APPROVED** [review]

**Comment by phofl** [comment]
thx @ambujpawar

**Pull request merged**